src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -122,24 +122,26 @@ const AuditorRunTestForm: React.FC = () => {
   };
 
   const confirmConnectModal = () => {
-    setTimeout(() => {
+    if (hasAnActiveSubscription) {
       confirm({
         title: "Verify the Repository details",
         description:
           "Unable to find the entered repository. Please go back and correct the Owner/Repository. Or, is this a Private one? If so, please hit Connect to authorize us to access it!",
         confirmationText: "Connect",
         cancellationText: "Go back",
       }).then(privateRepoDisclaimer).catch(err => {});
-    }, 0);
+    }
   };
 
   const privateRepoDisclaimer = () => {
-    confirm({
-      title: "Private Repository Access Disclaimer",
-      description:
-        "Auditors need to obtain consent from their customers and acquire the necessary permissions to fork their private Github repositories in order to test the decentralized application (dApp) using the Plutus Testing Tool, created by Input Output Global, Inc (IOG). The Plutus Testing Tool is available on an “AS IS” and “AS AVAILABLE” basis, without any representation or warranties of any kind. IOG is not responsible for the actions, omissions, or accuracy of any third party for any loss or damage of any sort resulting from the forking of repositories and testing of dApps using the Plutus Testing Tool.",
-      confirmationText: "Agree",
-    }).then(connectToGithub).catch(err => {});
+    if (hasAnActiveSubscription) {
```

Comment:
If already checked for hasAnActiveSubscription in confirmConnectModal, then this can be avoided right

Reply:
Fixed!